seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlIT.java

```diff
@@ -194,11 +196,14 @@ Pair<String[], List<SeaTunnelRow>> initTestData() {
                 };
 
         List<SeaTunnelRow> rows = new ArrayList<>();
+        // Define a large bigint data value to test for compatibility above bigint(20)
+        BigDecimal testBigDecimalValue = new BigDecimal("2844674407371055000");
```

Comment:
The number 2844674407371055000 is within the storage range of a long. You should use a value that is outside of long's storage capacity, such as 899,999,999,999,999,999,999,999,999,999.